resources/views/layouts/navigation.blade.php

```diff
@@ -20,25 +20,22 @@
                         {{ __('Game List') }}
                     </x-nav-link>
 
-                    <div class="mb-2 mt-3 ml-3 xl:w-96">
+                    <div class="mb-2 mt-3 ml-3 ">
                         <div class="input-group relative flex flex-wrap items-stretch w-full mb-4">
-                            <form method="POST" action="{{ route('results') }}">
+                            <form method="POST" class="inline-flex" action="{{ route('results') }}">
                                 @csrf
 
                                 <input type="search" class="form-control relative flex-auto min-w-0 block w-full px-3 py-1.5 text-base font-normal text-black-700 bg-white bg-clip-padding border border-solid border-gray-500 rounded transition ease-in-out m-0 focus:text-black-700 focus:bg-white focus:border-blue-600 focus:outline-none" placeholder="Search" aria-label="Search" aria-describedby="button-addon2">
 
-                                
-                                    <button type="submit" class="btn inline-block px-6 py-4 bg-black-600 text-black font-medium text-xs leading-tight uppercase rounded shadow-md hover:bg-green-300 hover:shadow-lg focus:bg-blue-700  focus:shadow-lg focus:outline-none focus:ring-0 active:bg-blue-800 active:shadow-lg transition duration-150 ease-in-out flex items-center" type="button" id="button-addon2">
+
+                                    <button type="submit" class="btn px-4 py-4 bg-black-600 text-black font-medium text-xs leading-tight uppercase rounded shadow-md hover:bg-green-300 hover:shadow-lg focus:bg-blue-700  focus:shadow-lg focus:outline-none focus:ring-0 active:bg-blue-800 active:shadow-lg transition duration-150 ease-in-out flex items-center" type="button" id="button-addon2">
```

Comment:
Good Style change. Inline-block removed.

Reply:
I was planning on working on the wishlist and after creating the branch I saw that Brian is working on it. I made these other changes and pushed them without realizing that the branch doesn't match.